cips/cip-31.md

```diff
@@ -0,0 +1,59 @@
+| cip           | xx                                                                                         |
+|---------------|--------------------------------------------------------------------------------------------|
+| title         | Add Hyperlane to celestia-app                                                              |
+| description   | This CIP proposes adding Hyperlane to celestia-app.                                        |
+| author        | c-node [@c-node](https://github.com/S1nus)                                                 |
+| discussions-to | [forum](https://forum.celestia.org/t/cip-add-hyperlane-bridging/1909)                     |                                                                     |
+| status        | Draft                                                                                      |
```

Comment:
IIUC this should actually be "Idea" based on: https://cips.celestia.org/cip-1.html#cip-process

Reply:
done